skbio/stats/composition.py

```diff
@@ -196,6 +202,9 @@ def multiplicative_replacement(mat, delta=None):
         delta = (1. / num_feats)**2
 
     zcnts = 1 - tot * delta
+    if np.any(zcnts) < 0:
+        raise ValueError('The multiplicative replacment created negative '
```

Comment:
Should this be a `UserWarning` or is it a proper error that should halt execution?